APIs/SweccAPI.py

```diff
@@ -60,4 +64,31 @@ async def reset_password(self, discord_username, id):
 
         response = requests.post(f"{self.url}/members/reset-password/", headers=self.headers, json=data)
         data = response.json()
-        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
\ No newline at end of file
+        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
+
+    async def process_reaction_event(self, payload, type):
+        user_id, channel_id, emoji = (
+            payload.user_id,
+            payload.channel_id,
+            payload.emoji,
+        )
+
+        if channel_id in self.reaction_channel_subscriptions and emoji.name == "✅":
```

Comment:
Maybe store this emoji as a constant somewhere in the class?

Reply:
done